frontend/src/app/components/containers/complaints/details/complaint-details-edit.tsx

```diff
@@ -181,13 +184,23 @@ export const ComplaintDetailsEdit: FC = () => {
   const [selectedIncidentDateTime, setSelectedIncidentDateTime] = useState<Date>();
   const [latitude, setLatitude] = useState<string>("0");
   const [longitude, setLongitude] = useState<string>("0");
+  const [linkedComplaintData, setLinkedComplaintData] = useState([]);
 
   const [complaintAttachmentCount, setComplaintAttachmentCount] = useState<number>(0);
 
   const handleSlideCountChange = (count: number) => {
     setComplaintAttachmentCount(count);
   };
 
+  //-- api calls
+  const getLinkedComplaints = async (complaintId: string) => {
```

Comment:
This is just my opinion but I think I'd typically like to see API calls be behind reducer actions unless they're very specifically internal to a component. It would probably make sense to create a getLinkedComplaints action and a selector for the linked complaint data? An argument for that is that other future components will also probably want to fetch and use the linked complaint data in our app.

Reply:
backlog item created